src/promptflow/tests/conftest.py

```diff
@@ -172,35 +172,35 @@ def side_effect(module_name, *args, **kwargs):
 
```

Comment:
Do we have stats of how this PR impact our live mode test duration?

Reply:
not yet, Philip's PR on collecting workflows stat is merged yesterday, so we can collect it since today; meanwhile, this PR will not affect too many tests as those fixtures (especially runs) are not widely used; I will open another PR to apply it widely, I think I can provide stats there.